src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px 0 rgba(0,0,0, 0.05);
+
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.logo {
+  &__link {
+    display: block;
+  }
+
+  &__image {
+    display: block;
+  }
+}
+
+.nav {
```

Comment:
Create a separate file for each BEM block

Reply:
Would be great to create a separate file for `nav` block too